ssl/quic/json_enc.c

```diff
@@ -602,10 +603,19 @@ void ossl_json_f64(OSSL_JSON_ENC *json, double value)
     if (!json_pre_item(json))
         return;
 
-    if (isnan(value) || isinf(value)) {
-        json_raise_error(json);
-        return;
+#if defined(__STDC_VERSION__) && __STDC_VERSION__ >= 199901L
```

Comment:
man page for isnan indicates that its only defined for _POSIX_C_SOURCE >= 200112L

Reply:
What man page are you reading ? I'm not saying you're *wrong*, but the following tells me more truths than just the POSIX indication:

https://linux.die.net/man/3/isnan
https://en.cppreference.com/w/c/numeric/math

Reply:
NM, I misread what you wrote.  I read 199901L As being 1990, which would have predated the isnan macro, but you have it right